[openmpi] build failure

**Host Environment**
- OS: Linux
- Compiler: GCC 9

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install openmpi`

**Failure logs** 
```
configure: WARNING: discovered external PMIx version is less than internal version 3.x
configure: WARNING: using internal PMIx
configure: WARNING: -g has been added to CFLAGS (--enable-debug)
configure: WARNING:  -fno-strict-aliasing has been added to CFLAGS
configure: WARNING: -g has been added to CXXFLAGS (--enable-debug)
configure: WARNING:  -finline-functions has been added to CXXFLAGS
configure: WARNING: -g has been added to CXXFLAGS (--enable-debug)
configure: WARNING:  -finline-functions has been added to CXXFLAGS
configure: WARNING: Open MPI now ignores the F77 and FFLAGS environment variables; only the FC and FCFLAGS environment variables are used.
configure: WARNING: -g has been added to FCFLAGS (--enable-debug)
**********************************************************************
* It appears that your Fortran compiler is unable to produce working
* executables.  A simple test application failed to properly
* execute.  Note that this is likely not a problem with Open MPI,
* but a problem with the local compiler installation.  More
* information (including exactly what command was given to the
* compiler and what error resulted when the command was executed) is
* available in the config.log file in the Open MPI build directory.
**********************************************************************
configure: error: Could not run a simple Fortran program.  Aborting.
```
**Additional context**
Why does it need Fortran in a C++ project? FFS OpenMPI. Can you please just deprecate it and switch to Intel MPI? It just doesn't work, and when it does, it breaks next month.

--- a/ports/openmpi/portfile.cmake
+++ b/ports/openmpi/portfile.cmake
@@ -15,7 +15,7 @@ vcpkg_extract_source_archive_ex(
     OUT_SOURCE_PATH SOURCE_PATH
     ARCHIVE ${ARCHIVE}
     PATCHES 
-	keep_isystem.patch
+        keep_isystem.patch
 )
 
 vcpkg_find_acquire_program(PERL)
@@ -28,6 +28,7 @@ vcpkg_configure_make(
         OPTIONS
             --with-hwloc=internal
             --with-libevent=internal
+            --disable-mpi-fortran
         OPTIONS_DEBUG
             --enable-debug
 )
@@ -37,4 +38,4 @@ vcpkg_install_make(DISABLE_PARALLEL)
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 
-file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

--- a/ports/openmpi/vcpkg.json
+++ b/ports/openmpi/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "openmpi",
   "version-string": "4.1.0",
-  "port-version": 1,
+  "port-version": 2,
   "description": "The Open MPI Project is an open source Message Passing Interface implementation that is developed and maintained by a consortium of academic, research, and industry partners. Open MPI is therefore able to combine the expertise, technologies, and resources from all across the High Performance Computing community in order to build the best MPI library available. Open MPI offers advantages for system and software vendors, application developers and computer science researchers.",
   "homepage": "https://www.open-mpi.org/",
   "supports": "!(windows | uwp)"

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4818,7 +4818,7 @@
     },
     "openmpi": {
       "baseline": "4.1.0",
-      "port-version": 1
+      "port-version": 2
     },
     "openmvg": {
       "baseline": "1.6",

--- a/versions/o-/openmpi.json
+++ b/versions/o-/openmpi.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "2c4b5b526fe4b6a8de9fa88eea30d3a1ddc7878e",
+      "version-string": "4.1.0",
+      "port-version": 2
+    },
+    {
       "git-tree": "bcc3284d0c3730ee4237efff0cd32df0be272d8e",
       "version-string": "4.1.0",
       "port-version": 1

